src/test/groovy/org/gradle/android/CrossVersionOutcomeAndRelocationTest.groovy

```diff
@@ -547,5 +547,15 @@ class CrossVersionOutcomeAndRelocationTest extends AbstractTest {
         builder.expect(':app:mergeReleaseResources', SUCCESS)
         builder.expect(':app:processDebugResources', SUCCESS)
         builder.expect(':app:processReleaseResources', SUCCESS)
+        // New tasks in 4.2.0-beta04
```

Comment:
Are these actually new in 4.2.0?  If so, why are they in the "android7xOrHigher" list?